src/core/hyprlock.cpp

```diff
@@ -698,9 +701,14 @@ static const ext_session_lock_v1_listener sessionLockListener = {
 // end session_lock
 
 void CHyprlock::onPasswordCheckTimer() {
+    static auto* const PNOFADEOUT  = (Hyprlang::INT* const*)g_pConfigManager->getValuePtr("general:no_fade_out");
```

Comment:
yes, but if XDG_CURRENT_DESKTOP != Hyprland we should force it to off.

Reply:
I'll make the changes, thanks!